specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/assessments.json

```diff
@@ -0,0 +1,368 @@
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "links": {
+          "$ref": "#/definitions/AssessmentLinks"
+        }
+      },
+      "required": [
+        "resourceDetails",
+        "status"
+      ]
+    },
+    "AssessmentLinks": {
+      "type": "object",
+      "description": "Links relevant to the assessment",
+      "readOnly": true,
+      "properties": {
+        "azurePortal": {
+          "type": "string",
+          "description": "Link to assessment in Azure Portal",
+          "readOnly": true
+        }
+      }
+    },
+    "AssessmentStatus": {
+      "type": "object",
+      "description": "The result of the assessment",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Programmatic code for the status of the assessment",
+          "enum": [
+            "Healthy",
+            "Unhealthy",
+            "NotApplicable"
+          ],
+          "x-ms-enum": {
+            "name": "AssessmentStatusCode",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Healthy",
+                "description": "The resource is healthy"
+              },
+              {
+                "value": "Unhealthy",
+                "description": "The resource has a security issue that needs to be addressed"
+              },
+              {
+                "value": "NotApplicable",
+                "description": "Assessment for this resource did not happen"
+              }
+            ]
+          }
+        },
+        "cause": {
+          "type": "string",
+          "description": "Programmatic code for the cause of the assessment status"
+        },
+        "description": {
+          "type": "string",
+          "description": "Human readable description of the assessment status"
+        }
+      },
+      "required": [
+        "code"
+      ]
+    }
+  },
+  "parameters": {
+    "ExpandAssessments": {
+      "name": "$expand",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "OData expand. Optional.",
+      "x-ms-parameter-location": "method",
+      "enum": [
+        "links",
+        "metadata"
+      ],
+      "x-ms-enum": {
+        "name": "ExpandEnum",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "links",
+            "description": "All links associated with an assessment"
+          },
+          {
+            "value": "metadata",
+            "description": "Assessment metadata"
+          }
+        ]
+      }
+    },
+    "AssessmentName": {
+      "name": "assessmentName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The Assessment Key - Unique key for the assessment type",
+      "x-ms-parameter-location": "method"
+    },
+    "AssessmentBody": {
+      "name": "body",
```

Comment:
Typically this is called "parameters".  The other security swagger specs use the resource type name (i.e. "assessment").  "body" isn't descriptive

Reply:
not sure what you meant here, can you please validate that the change is correct?